app/uk/gov/hmrc/gform/eval/EvaluationResults.scala

```diff
@@ -112,7 +111,12 @@ case class EvaluationResults(
       case ctx @ FormCtx(formComponentId: FormComponentId) => get(ctx, recData, fromVariadicValue)
       case Sum(FormCtx(formComponentId)) =>
         calculateSum(formComponentId, recData, unsupportedOperation("Number")(expr))
-      case Sum(_)                                     => unsupportedOperation("Number")(expr)
+      case Sum(_) => unsupportedOperation("Number")(expr)
+      case Count(formComponentId) =>
+        val xs: Iterable[(ModelComponentId, VariadicValue)] =
+          recData.variadicFormData.forBaseComponentId(formComponentId.baseComponentId)
+        val zeros: Int = xs.map(_._2).count(_.contains(0.toString))
+        NumberResult(zeros + 1)
```

Comment:
Can we please look into code formating for better reading?

Reply:
This is `scalafmt` formatted. What exactly is your concern?

Reply:
<img width="959" alt="Screen Shot 2021-02-02 at 09 53 41" src="https://user-images.githubusercontent.com/17478517/106583221-c3a8e100-653c-11eb-969c-e9724b23cb9a.png">

wouldn't be easy to read at first instance.

Reply:
Fixed